src/main/resources/messages.properties

```diff
@@ -13,9 +13,15 @@ ExceedTotal.courseConfigurationForm.defaultMinutesPerSession = Must not exceed t
 typeMismatch.makeupForm.entries.minutesMadeUp = Must be a valid number.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the whitespaces between the key and the separator. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aseparator-convention)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the whitespaces between the separator and the value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aseparator-convention)